global.json

```diff
@@ -1,6 +1,6 @@
 {
   "sdk": {
-    "version": "8.0.101",
+    "version": "8.0.202",
```

Comment:
This isn't a safe change, unfortunately. We need to make sure we stay below what's available in App Service.

Reply:
Reverted - but being a patch version change, I wonder if it would be a problem? Would an app build on this SDK (thus `8.0.2` runtime) still be runnable by `8.0.1` runtime?

After all, we can build net6.0 apps with `8.0.x` SDK and run it no problem

Reply:
In theory, that should be the case, but we do have evidence of issues (including with current/recent patches).